---
title: "Part 2: Methods in critical review "
file: "part2_methods.qmd"
bibliography: references.bib
---

\pagebreak

## Introduction

In the two studies included in this thesis, we explore potential mediators of the effect of pharmacological SGLT2 inhibition on risk of cardiovascular disease and mortality. In **Paper I**, we use data from large population studies in Denmark, the United Kingdom, and Finland, and explore the relationship between genetic variation in *SLC5A2*, encoding SGLT2, and risk of heart failure, myocardial infarction, ischemic heart disease, ischemic stroke, cardiovascular mortality, and all-cause mortality and aim to quantify the proportion of the total effect mediated through different potential mediators. In **Paper II**, we meta-analyzed the results of 60 randomized placebo-controlled trials of SGLT2 inhibitors to investigate the effect of treatment on levels of total, LDL, and HDL cholesterol and plasma triglycerides, all known to play a role in atherosclerosis and subsequently in cardiovascular disease and risk of premature death.\
In the following part of the thesis, methods and materials used in the two original papers will be expanded on and a variety of biases and limitations that may have affected the results will be discussed. Some biases, we can quite safely dismiss, while the influence of others is not easily quantified.\
For more specific descriptions of methods, please refer to the enclosed papers and their supplementary appendices.

In @tbl-methods an overview of the methods and materials used in the two papers is shown.

\pagebreak

+---------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
|               | **Paper 1**                                                                                                                                                             | **Paper 2**                                                                                                                                                                 |
|               |                                                                                                                                                                         |                                                                                                                                                                             |
|               | *Genetic variation in SLC5A2 mimicking SGLT2-inhibition and risk of cardiovascular disease and all-cause mortality: reduced risk not explained by lower plasma glucose* | *SGLT2-inhibition increases total, LDL, and HDL cholesterol and lowers triglycerides: meta-analyses of 60 randomized trials, overall and by dose, ethnicity, and drug type* |
+---------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| **Exposure**  | *SLC5A2* genetic variation                                                                                                                                              | SGLT2-inhibitor treatment                                                                                                                                                   |
+---------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| **Endpoints** | Heart failure\                                                                                                                                                          | Change in:\                                                                                                                                                                 |
|               | Myocardial infarction\                                                                                                                                                  | Total cholesterol\                                                                                                                                                          |
|               | Ischemic heart disease\                                                                                                                                                 | LDL cholesterol\                                                                                                                                                            |
|               | Ischemic stroke\                                                                                                                                                        | HDL cholesterol\                                                                                                                                                            |
|               | Cardiovascular mortality\                                                                                                                                               | Triglycerides                                                                                                                                                               |
|               | All-cause mortality                                                                                                                                                     |                                                                                                                                                                             |
+---------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| **Materials** | CCHS: N = 10,344\                                                                                                                                                       | 60 randomized controlled trials, including 147,130 individuals                                                                                                              |
|               | CGPS: N = 102,368\                                                                                                                                                      |                                                                                                                                                                             |
|               | UK Biobank: N = 488,687\                                                                                                                                                |                                                                                                                                                                             |
|               | FinnGen: N = 342,499                                                                                                                                                    |                                                                                                                                                                             |
+---------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| **Methods**   | Survival analysis\                                                                                                                                                      | Meta-analysis                                                                                                                                                               |
|               | Observational analysis\                                                                                                                                                 |                                                                                                                                                                             |
|               | Mediation analysis                                                                                                                                                      |                                                                                                                                                                             |
+---------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------+

: Overview of the main materials and methods used in the two papers included in the thesis. CCHS = the Copenhagen City Heart Study, CGPS = the Copenhagen General Population Study. {#tbl-methods}

\pagebreak

## Study populations and materials

The two original papers are based on population data and published randomized controlled trials, respectively. **Paper I** is based on population data from the Copenhagen City Heart Study and the Copenhagen General Population Study, and validated with data from the UK Biobank and FinnGen. **Paper II** is based on results from 60 different randomized placebo-controlled trials of SGLT2 inhibitors.

### Paper I

#### The Copenhagen City Heart Study

The Copenhagen City Heart Study is a study of the Danish general population that began in 1976-1978, with subsequent follow-up examinations conducted in 1981-1983, 1991-1994, and 2001-2003, designed to investigate the occurrence and patterns of cardiovascular disease. Eligible participants were invited randomly from a pool of approximately 90,000 individuals of Danish descent residing near Rigshospitalet, Copenhagen University Hospital, using the Danish Civil Registration System. Invitations to participate were stratified by sex and 5-year age groups to ensure an accurate representation of the adult population aged 20-80+ years. Prior to examination, participants were provided with a comprehensive questionnaire covering various aspects of their health and lifestyle, which they were required to complete. On the day of the examination, the questionnaire answers were reviewed by an examiner. Participants underwent physical examination which included measurements of blood pressure, height, weight, and other relevant factors, and had blood samples drawn for biochemical analyses and biobanking. In the third and fourth round of examinations, participants were also asked to provide consent for DNA extraction, leading to the establishment of a DNA biobank [@nordestgaard1996]. The initial examination included a total of 14,223 individuals. For the second examination, the cohort was supplemented with 500 men and women aged 20-24 years. In the third examination, an additional 3,000 men and women aged 20-49 years (250 men and 250 women in each 5-year age group) were included, while the fourth examination involved the inclusion of an additional 1,040 individuals aged 20-34 years. The participation rate was 72% in 1976-1978[@appleyard]. All individuals who were originally invited to the study, including the supplemental participants, were invited (if still alive) to attend the follow-up examinations, regardless of their current address outside the study area. The response rates for the follow-up examinations were 70% (1981-1983), 61% (1991-1994), and 50% (2001-2003), with the third examination involving the participation of 10,049 individuals and 9,259 agreeing to contribute blood samples to the DNA biobank. The fourth examination included 6,237 participants, all agreeing to provide blood samples for the DNA biobank [@nordestgaard1996]. 99.5% of the participants were of white ethnicity and of Danish descent. The study was approved by institutional review boards and Danish ethical committees (KF-100.2039/91, KF-01-144/01, H-KF-01-144/01) and was conducted according to the declaration of Helsinki. Written informed consent was obtained from all individuals.

For **Paper I,** baseline data from the first examination of each individual (examinations 1-4) was used for the analyses; however, depending on the participants having been examined in examination 3 or 4 in order to collect DNA for genetic analyses. All 10,344 individuals with available genetic information for the study were included.

#### The Copenhagen General Population Study

The Copenhagen General Population Study is an ongoing cohort study that was initiated in 2003-2015. Follow-up examinations began in 2014 and are ongoing. Participants were invited from various geographical regions within the greater Copenhagen area, with 25% of individuals aged 20-39 years and 100% of individuals aged over 39 years invited to participate. The participation rate for the first examination was 42%. Procedures for questionnaire, physical examination, blood sampling for biochemical analyses, DNA, and biobanking were similar to the Copenhagen City Heart Study. Similar to the Copenhagen City Heart Study, 99.5% of the participants were of white ethnicity and of Danish descent. The study was approved by institutional review boards and Danish ethical committees (KF-100.2039/91, KF-01-144/01, H-KF-01-144/01) and was conducted according to the declaration of Helsinki. Written informed consent was obtained from all individuals. Furthermore, no individuals were included in both the Copenhagen City Heart Study and the Copenhagen General Population Study, as potential overlaps were removed before statistical analysis. Because of the similarity between the Copenhagen City Heart Study and the Copenhagen General Population Study, the studies were analyzed together in **Paper I** (the Copenhagen City Heart Study + the Copenhagen General Population Study), and are sometimes referenced to as the Copenhagen studies.

For **Paper I,** all 102,368 individuals with available genetic information for the study were included.

#### The UK Biobank

The UK Biobank is a large prospective study of the middle and old age general population of the United Kingdom. The study was initiated in 2005-2010 with the aim of providing a large resource for researchers to perform health research in the public interest. More than 500,000 individuals aged 40-69 at the time of entry were included. Participants were selected using the National Health Service (NHS) register and invited to volunteer for the study. Information on characteristics and phenotypic and genotypic traits of the participants has been continuously collected. This includes data obtained from questionnaires, physical measurements, blood samples for biochemical and genetic analyses, accelerometry, multimodal imaging, genome-wide genotyping, and longitudinal follow-up for a diverse array of health-related outcomes using national health registries[@sudlow2015; @bycroft2018]. The participation rate for the initial examination was 5%[@fry2017].\
The study was approved by the North West Haydock Research Ethics Committee(16/NW/0274).

For **Paper I,** 488,687 individuals with available genetic information for the study were included. Among these, 460,493 identified as being white, 9,466 as Asian, 7,638 as Black, 2,870 as mixed ethnicity, 1,503 as Chinese, and 6,717 did not report on ethnicity.

#### FinnGen

FinnGen is a Finnish study initiated in 2017 aiming to include 500,000 individuals. It is a public-private partnership with the aim of gaining novel insights into disease genetics by combining imputed genotype data from Finnish biobanks with digital health record data sourced from Finnish health registries. The initiative includes 9 Finnish biobanks, research institutes, universities, university hospitals, 13 international pharmaceutical industry partners, and the Finnish Biobank Cooperative (FINBB). The project makes use of data from the nationwide longitudinal health register collected since 1969 from every resident in Finland. The Finnish population is a population isolate which means that some genetic variants are enriched due to founder effects. At the time of data collection for **Paper I**, genotyping and phenotyping information was available on 342,499 individuals (release 8). Participants had a median age of 63 years and were predominantly recruited at hospitals[@kurki2023].

For **Paper I,** summary level endpoint data from release 8 from December 2022 was used.

#### Limitations and risk of bias in study populations

The Copenhagen City Heart Study, the Copenhagen General Population Study, and the UK Biobank were all designed to be studies of the general population. This means that the distribution of invitations to participate in the studies was designed to include and reflect the general population, making it reasonable to extrapolate results from studies on the cohorts to the actual general population. However, to fulfill this, first, the general characteristics of the invited participants should match the actual general population in respect numerous baseline characteristics, such as age, sex, ethnicity and/or race, socioeconomic status, and disease risk factors. Second, all invited participants should then choose to take part in the study. Deviations from these requirements may give rise to a range of different selection biases.\
A fundamental point to address in genetic population studies, is the Hardy-Weinberg equilibrium, stating that allele frequencies in a population are in equilibrium[@lachance2016]. This equilibrium may be disturbed by a number of factors, however, most often deviations are a sign of genotyping errors. Other influences include genetic drift, non-random mating (see "The 'genetic variation as proxy for drug effects' study design in Paper I" section), or the founder effect. If Hardy-Weinberg expectations are not met, first genotyping errors must be ruled out, next, potential biases in the study population must be considered, as these may also give rise to deviations.

Some of the biases that may influence our results are:

-   **Sampling bias or ascertainment bias** may occur when members of the intended population (in this case the general population) are less likely to be invited or included compared to others. In both the Copenhagen City Heart Study and the Copenhagen General Population Study, participants were invited from urban areas in and around Copenhagen, reducing the likelihood of a true reflection of the Danish general population.

-   **Undercoverage bias** may occur when some groups of the population are not or less represented in the sample. For example, the lack of invitations to participants of other ethnicities and/or race than Danish/white in the Copenhagen City Heart Study and the Copenhagen General Population Study means that the cohorts do not in fact reflect the Danish general population, which consists of 15.4% immigrants or descendants of immigrants[@immigran], and that results from studies in these populations cannot be extrapolated to include non-ethnic Danish individuals of the Danish general population.

-   **Non-response bias** may occur when individuals who choose not to participate systematically differ from the ones who do participate. Participation rates were 50-72% and 42% in the Copenhagen City Heart Study and the Copenhagen General Population Study, respectively, and 5% in the UK Biobank, indicating a risk of non-response bias. This risk is especially pronounced for the UK Biobank which had the lowest participation rate. It has been shown that non-responders differ significantly from responders in population cohorts with higher rates of morbidity and mortality in non-responders[@ferrie2009; @larsen2012]. A large number of non-responders may skew the results on risk estimates toward the null-hypothesis and might therefore influence the external validity or the generalizability of the results. However, the internal validity, meaning the general trustworthiness of the relationship under examination, is not influenced unless non-response, or other selection biases, are associated with both the exposure and outcome.

-   **Attrition bias** may occur when participants who drop out of a study systematically differ from those who remain. For **Paper I**, DNA samples were only available for individuals in the Copenhagen City Heart Study who had been included in examination 3 or 4, meaning that either participants had to be newly enrolled, or they had to have not dropped out of the study until they reached examination 3 or 4 in order to be included in the analyses in the paper. Attrition bias might therefore affect the results in **Paper I**; however, we found that the *SLC5A2* rs61742739 genetic variant used for the analyses was in Hardy-Weinberg equilibrium in both the Copenhagen City Heart Study, the Copenhagen General Population Study, and the UK Biobank, substantiating the notion that the genotypic composition of the cohorts is not skewed due to for example attrition bias.\
    The Copenhagen City Heart Study, the Copenhagen General Population Study, and the UK Biobank all expand their data collection through follow-up examinations, with the potential of declining participation rates (see above for the Copenhagen City Heart Study response rates for the follow-up examinations); however, in **Paper I**, we only used data from the participants' first visit (except for genetic information, which could have been collected at a later time point).

-   **Healthy volunteer/participant bias** may occur when the individuals who choose to participate are systematically healthier than those who do not. In the UK Biobank, this is for example demonstrated by lower rates of obesity, smoking, alcohol intake and self-reported health conditions compared to the general population of the United Kingdom of the same age. Also, women, older individuals, and those residing in less socioeconomically deprived areas had higher rates of participation in the UK Biobank[@fry2017].

-   **Survivor bias or immortal time bias** may occur when only participants who have survived past a certain time point can be part of the study, meaning that included participants have been "immortal" up until initiation of the study or follow-up. In **Paper I** we argue that it makes sense to follow participants from the earliest time point possible, because genotypes do not change during the course of life. However, in order to be part of the study, all individuals in the Copenhagen City Heart Study had to survive until examination 3 or 4 to be enrolled in the population for **Paper I**, possibly giving rise to survivor bias, although the original cohort was supplemented with young individuals at the three follow-up examinations to mitigate this risk. For all cohorts, participants had to be alive to be invited. However, as for attrition bias, it is unlikely that this has affected the population to a major degree as the *SLC5A2* rs61742739 genotype was in Hardy-Weinberg equilibrium in the Copenhagen City Heart Study, the Copenhagen General Population Study, and the UK Biobank.

-   **Loss to follow-up** is a potential cause of bias in longitudinal studies, as there may be differences between individuals lost to follow-up and the ones who remain in the cohort. Because we use register-data for the endpoints, and because the structure of the registers ensures follow-up of all individuals except for those who emigrate (\<1% in the Danish cohorts), the loss to follow-up is very low. It is unlikely that the small proportion lost due to emigration differ significantly from the rest of the cohort and that this would bias the estimates to a major degree.

In **Paper I**, summary level data on the genotype and endpoints from FinnGen was used for validation. This approach is limited by the lack of individual level data and especially the different composition of FinnGen versus the Copenhagen studies. As mentioned, the Finnish population, being a population isolate, may provide several opportunities in genetic research because of the enrichment of certain variants. Conversely, the genetic isolate effect may also cause some variants to be extremely rare due to being missing or similarly rare in the founding population. This will inevitably limit the interpretation of some genetic variant studies, that will simply be underpowered compared to a similar study in another genetic population. For the *SLC5A2* rs61742739 genotype, only 5 homozygotes were found among the 342,499 included individuals[@pheweb], and the reported minor allele frequency was 0.0018, noticeably lower than the minor allele frequencies of 0.01 in the Copenhagen studies and the UK Biobank. Furthermore, the participants in FinnGen were predominantly recruited at hospitals, and is therefore enriched for disease endpoints compared to a general population. Together these differences might cause an underestimation of the genotype effect, potentially limiting the use of FinnGen data as validation data for general population results.

### Paper II

#### Randomized controlled trials of SGLT2 inhibitors

For **Paper II**, the material was based on randomized controlled trials of SGLT2 inhibitors reporting on change in lipids and lipoproteins. Different from the population studies described above, participants in randomized controlled trials are selected because of their specific characteristics and not as representatives of the general population. For the majority of the studies included, participants had a diagnosis of diabetes, since SGLT2 inhibitors were initially only indicated for treatment of this specific patient group. This must of course be considered when interpreting the results, since it is expected that lipid metabolism is altered in the diabetic state.\
For the meta-analysis the studies were screened based on an extensive literature search and inclusion criteria were:

-   Studies had to be randomized placebo-controlled trials on SGLT2 inhibitors and have available lipid- and lipoprotein measurements as either mean change from baseline or baseline and follow-up measurements including standard deviation (SD), standard error (SE), or 95% confidence intervals (95% CI)

-   Total N \>=50

-   Treatment duration \>= 4 weeks

Exclusion criteria were:

-   If the tested drug was a non-selective SGLT2 inhibitor (such as sotagliflozin which also acts as an SGLT1-inhibitor)

-   Lack of available lipid measurements or if measures were reported only as median (interquartile range) or percentage change from baseline

The first step of the meta-analysis, after forming the hypothesis to be tested and defining inclusion and exclusion criteria, was the literature search to start the process of collecting the necessary material for analysis. At the literature search state of any study, considering and applying a search strategy is advised. A commonly used strategy is the Population, Intervention, Comparison, Outcome (PICO) to identify the correct studies and limit the amount of irrelevant search hits[@agoritsas2012]. Most research search engines only search in abstracts of published papers, making researchers dependent on abstracts to contain the PICO information for this strategy to work. With the growing body of evidence and published medical literature, the call for effective search algorithms to retrieve the results that they are actually looking for is more important than ever. It is increasingly more common to publish open access and to include the full text of a paper in PubMed Central, making it to searchable in PubMed, and while this provides better opportunities to find what you are looking for [@kim2022], it also comes with a risk of bias since open access publishing is not equal for all. When online publishing began to overtake as the main source of scientific information in the 2000's, the paywalls to read papers made the inequality in accessibility very visible, spurring the call for open access. The pursuit of open access has led to a situation where publishers have shifted to supply-side funding, which means that only those with access to funding are able to share their research, continuing the practice of inequality in accessibility to publishing and reading scientific papers[@green2019]. All this means that even though all research that is published today is available online - with or without a paywall - it is still not possible to conduct an actual full-text search to find the exact details one might be looking for. When an entire project depends on the search strategy, such as a meta-analysis, this problem can become very time-consuming to solve and may even cause faulty results or valid projects to be abandoned early on. In **Paper II**, we were interested in measurements of lipids and lipoproteins that were rarely primary endpoints of the randomized controlled trial but rather secondary endpoints or part of safety parameters, maybe or maybe not reported in the paper and almost never in the abstract. To mitigate the risk of excluding papers with available lipid information, PubMed, EMBASE, and Web of Science were searched for all randomized placebo-controlled trials on all SGLT2 inhibitors written in English with no inclusion of lipids or lipoproteins in the search term. This was done to make sure that all studies reporting changes in total cholesterol, LDL cholesterol, HDL cholesterol, and triglycerides could be included, even if these search words were not mentioned in the abstract. The search strategy led to 2,346 unique abstracts. After screening, 419 papers were found eligible for full-text screening or full-text search of search words matching the endpoints. This was done in the statistical program R, using regular expressions, where a list of all papers with one or more matches was returned. The papers were then read to do the final screening, and in the end, 60 studies with available information in change in total cholesterol, LDL cholesterol, HDL cholesterol, and triglycerides were included (@fig-prisma).

![Flow chart of the studies identified, included, and excluded in the meta-analyses in **Paper II**[@bechmann2023a].](prisma.png){#fig-prisma}

#### Limitations and bias in randomized controlled trials of SGLT2 inhibitors

The effort to identify all published studies of SGLT2 inhibitor randomized placebo-controlled trials reporting on change in lipids and lipoproteins could potentially have been affected by a number of different biases and limitations.

-   **Selection bias** may occur if the search strategy is insufficient to identify all studies or in the manual process of screening and selecting abstracts for full-text screening. This may have led to relevant studies not being included. However, since the search strategy was designed to minimize this risk (as described above), it is unlikely that selection bias severely affects the results of the meta-analysis.

-   **Publication bias** may occur when the publication of a study depends on the results and has the potential to critically bias meta-analyses. There is convincing evidence that studies reporting positive findings are much more likely to be published than negative or null-findings[@schmucker2014; @scherer2018; @rising2008]. This can be due to both the publisher, who knows that positive findings receive more citations than negative or null-findings, or to the knowledge of the authors, that publishing negative or null-findings can be difficult and might not be worth the effort.

-   **Selective non-reporting bias** may occur when results are reported for some outcomes or analyses, but not others, depending on the nature and direction of the results. In the full-text screening process this issue arose often and several studies were excluded because a comment as: "No significant change was observed for cholesterol or triglycerides" was reported instead of the nominal change. In **Paper II**, 13 studies were excluded because of insufficient statistics reported, and 254 studies were excluded because of insufficient lipid measurements.

The issue of unavailable data is a serious limitation to studies of non-primary endpoints. In the case of **Paper II**, we estimate that approximately 270 studies, that were not already included and had the correct study design, were excluded because of missing data - either because change in lipids or lipoproteins were not measured as part of the study or because it was not reported in the published paper. This means that the data available for analysis is approximately 18% of the potential amount of data on change in lipids and lipoproteins in SGLT2 inhibitor randomized placebo-controlled trials.\
In meta-analyses, Begg and Mazumdar´s rank test and funnel plots are usually used to investigate potential publication- and selective non-reporting bias. In **Paper II**, no indications of publication and selective non-reporting bias was found using these tests; however, knowing there is a possibility that only 18% of eligible data was included, these tests only serve to show that both positive and negative changes of lipids and lipoproteins were reported, not that there was no publication- or selective non-reporting bias. If the reason for non-reporting bias in the remainder of the studies was no detected effects or null-findings, then we would expect that the results of the analyses overestimate the true effects.

## Selection and genotyping of genetic variants in Paper I

For **Paper I**, we used a functional genetic variant rs61742739 in the *SLC5A2* gene, encoding SGLT2, to test the hypothesis that genetic variation in *SLC5A2* mimics pharmacological SGLT2 inhibition.\
Participants in The Copenhagen City Heart Study and the Copenhagen General Population Study were genotyped for this variant in *SLC5A2,* located on chromosome 16 (@fig-slc5a2). The variant was selected from Ensembl (ensembl.org), The Online Mendelian Inheritance in Man (OMIM), the Human Gene Mutation database (HGMD), genome wide association studies (the Meta-Analyses of Glucose and Insulin-related traits consortium (MAGIC), the DIAbetes Genetics Replication And Meta-analysis Consortium (DIAGRAM), and available literature. Selection criteria were a minor allele frequency\>0.01 and/or association with glycemic traits and/or glucosuria in the form of familial renal glucosuria[@ordelheide2017; @zimdahl2017; @enigk2011; @drexel2019; @calado2004].

![The *SLC5A2* genetic variant rs61742739 genotyped in the Copenhagen City Heart Study and the Copenhagen General Population Study. The variant is located in the *SLC5A2* gene on chomosome 16.](slc5a2rs61.png){#fig-slc5a2}

More than 80 genetic variants in *SLC5A2* were described to cause or possibly cause familial renal glucosuria in OMIM, HGMD, and available literature. Of these, seven variants were reported both in OMIM, HGMD, and in the literature and six had minor allele frequencies below 0.001 in European reference populations. Rs61742739 was the most common variant with a minor allele frequency of 0.01 and also the variant most often reported in published studies to be associated with familial renal glucosuria[@santer2003; @calado2004; @kleta2004; @calado2006; @calado2008], and thus the variant we selected for **Paper I**. None of the rare variants (minor allele frequency\<0.001) were genotyped in the Copenhagen City Heart Study and in the Copenhagen General Population Study, since we do not have a sufficient number of individuals in these cohorts to identify enough carriers for sufficiently powered statistical analyses. These were not genotyped on the arrays in the UK Biobank or in FinnGen either.\
Familial renal glucosuria is a phenotype that resembles the effect of pharmacological SGLT2 inhibition. The minor allele G of rs61742739 alters the amino acid sequence, substituting asparagine with serine in position 654 of the SGLT2 protein[@santer2003; @calado2004; @kleta2004; @calado2006; @calado2008], and is known to be associated with familial renal glucosuria. The amino acid substitution is in a highly conserved position of the SGLT2 protein, indicating that this part of the protein is functionally important.\
In the Copenhagen City Heart Study and the Copenhagen General Population Study, genotyping of the genetic variants was done by TaqMan allelic discrimination assays using an ABI PRISM 7900HT Sequence Detection System (Applied Biosystems Inc., Foster City, CA) and genotypes were called using a 7600HT TaqMan system from Applied Biosystems, and subsequently verified manually by a lab technician. The TaqMan genotyping method is a polymerase chain-reaction (PCR)-based technique. Similar to conventional PCR methods, two primers that bind to the regions flanking the target DNA sequence during the PCR cycle are added to the DNA sample. With the assistance of a DNA polymerase, the DNA sequence of interest is replicated according to the binding of the primers. Two probes consisting of allele specific nucleotide sequences corresponding to the wildtype and minor allele of the target DNA sequence are included in the assay. Each probe is labeled with a distinct fluorescent dye at the 5' end and a fluorescent quencher at the 3' end. The quencher prevents fluorescence emission from the probe. Throughout the PCR cycle, the probe anneals to the DNA sequence, and as the DNA polymerase replicates the DNA, it cleaves the probe, releasing the fluorescent dye from the quencher, which results in fluorescence from the attached reporter dye. The wildtype and mutant alleles are associated with different fluorescent signals, allowing for the determination of the genotype based on the relative signal emitted by the cleaved probes at the end of the PCR cycle. Homozygous samples emit only one color, whereas heterozygous samples emit colors of both the wildtype and the minor allele[@essentia]. The genotyped variant was in Hardy-Weinberg equilibrium, supporting a low level of genotyping errors.

In the UK Biobank and FinnGen genotyping was done by Affymetrix/ThermoFisher Axiom custom arrays[@bycroft2018; @kurki2023]. Again, in the UK Biobank, the genotyped variant was in Hardy-Weinberg equilibrium, supporting a low level of genotyping errors. Because only summary data was available to us from FinnGen, Hardy-Weinberg equilibrium could not be assessed in FinnGen.

## Potential mediators and covariates in Paper I

In the Copenhagen City Heart Study and the Copenhagen General Population Study, participants completed a questionnaire, underwent physical examination, and had blood samples drawn for biochemical analyses and DNA samples. In the UK Biobank the process was similar, also involving a questionnaire, physical examination, and blood samples.\
In this section, the measurements used for the analyses in **Paper I** are described.

### The Copenhagen City Heart Study and the Copenhagen General Population Study

#### Self-administered questionnaire

Self-reported variables smoking (never, former, current) and physical activity (low, moderate, high) were used as covariates in **Paper I**. The variables history of diabetes mellitus, glucose-lowering drug use, and antihypertensive drug use were used together with the World Health Organization's International Classification of Diseases (ICD) codes to define the categorical covariates diabetes mellitus and hypertension.\
Using self-reported information on disease outcomes with the risk of recall bias, which occurs when the participant does not accurately recall a certain event. Diabetes and hypertension variables were only used for exploratory analyses, and recall bias is therefore not likely to affect the results of **Paper I**.

#### Physical examination

Body weight, height, systolic and diastolic blood pressure, were measured by trained technicians during the physical examination and were investigated as potential mediators of the effect of SGLT2 genetic variation on risk of cardiovascular disease and mortality in **Paper I**.

#### Biochemical analyses

Participants in the Copenhagen City Heart Study and the Copenhagen General Population Study had blood samples drawn at the day of attendance at a random timepoint since last meal.

-   Non-fasting plasma glucose was measured by colorimetric assays (Konelab, Thermo Fischer Scientific)

-   Plasma creatinine, sodium, and potassium were measured using standard hospital assays (Konelab, Thermo Fisher Scientific)

-   Blood hematocrit was measured using a flow cytometer‐based hematology analyzer (ADVIA^TM^120, Siemens)

-   Plasma total cholesterol, high-density lipoprotein cholesterol, and triglycerides were measured by direct enzymatic methods (Thermo Fisher Scientific/Konelab)

-   Plasma low-density lipoprotein (LDL)-cholesterol was calculated using the Friedewald equation if plasma triglyceride levels were ≤4.0 mmol/L, and otherwise measured using a direct assay

-   Lipoprotein(a) was measured using assays from DiaSys (Diagnostic Systems) and Denka Seiken and subsequently adjusted for assay differences

Sampling, measurements, handling, and storage of the samples were done with minimal variation over the years to ensure that results were comparable across time. Internal (daily) and external (ten times a year) quality control to ensure accuracy and reproducibility for all biochemical analyses were carried out continuously over the years.

Non-fasting plasma glucose, as opposed to fasting glucose, was used as a potential mediator/covariate in **Paper I**. Non-fasting glucose levels vary depending on time since last meal, which was recorded in data collection, but not adjusted for in the present analyses. However, non-fasting glucose has previously been shown to be more strongly associated with risk of cardiovascular disease than fasting glucose[@hanefeld1996], and it can be used in risk stratification[@benn2012].

LDL cholesterol was partly calculated using the Friedewald equation. In the recent years, other equations to calculate LDL cholesterol have been shown to give better estimates, especially in patients with low LDL cholesterol or high triglyceride levels[@sampson2020]. Direct measurement of LDL cholesterol is generally preferred in individuals with high triglyceride levels and may also be preferred in conditions as metabolic syndrome and diabetes, where the presence of cholesterol-poor small dense LDL particles can lead to a divergence between plasma LDL cholesterol and LDL particle concentration, making LDL cholesterol a less reliable measure for LDL particle concentration than usual[@ference2017]. Direct measurement was used for triglycerides ≤4.0 mmol/L, but conditions as metabolic syndrome and diabetes were not taken into account in our LDL cholesterol analyses, however, doing so is unlikely to change the results of **Paper I**, where LDL cholesterol was not shown to be associated with *SLC5A2* rs61742739 genotype.

### The UK Biobank

-   Non-fasting plasma glucose levels were determined using hexokinase analysis on the Beckman Coulter AU5800 system.

-   Hemoglobin A1c levels were measured by HPLC analysis using the Bio-Rad VARIANT II Turbo instrument.

-   Weight and height measurements were obtained during the initial Assessment Centre visit using various methods.

-   Systolic and diastolic blood pressure were obtained during the initial Assessment Centre visit using various methods.

-   Plasma creatinine levels were determined using enzymatic analysis on the Beckman Coulter AU5800 system.

-   Blood hematocrit was calculated as the relative volume of packed erythrocytes to whole blood using the formula: (red blood cells x mean corpuscular volume)/10 on the Beckman Coulter LH750 analyzer.

-   LDL cholesterol levels were measured directly using enzymatic protective selection analysis on the Beckman Coulter AU5800 system.

-   HDL cholesterol levels were measured by enzyme immunoinhibition analysis on the Beckman Coulter AU5800 system.

-   Triglyceride levels were measured by GPO-POD analysis on the Beckman Coulter AU5800 system.

-   Lipoprotein(a) levels were measured using immunoturbidimetric analysis on the Randox AU5800 system.

-   Plasma sodium and potassium measurements were not available in our UK Biobank data.

To ensure the reliability and accuracy of the biochemical analyses, performance and quality control procedures were followed. For each biomarker, participant samples were analyzed alongside Internal Quality Control (IQC) samples of known concentrations (high, medium, and low). Participant results were considered valid if both the opening and closing IQC results fell within the predetermined control limits for the analytical process. Third-party IQC materials were employed for each assay to provide an unbiased assessment of the overall analytical system's performance

### Risk of bias and limitations in potential mediators and covariates

Measurement error (continuous variables) or misclassification (categorical variables) has the potential to reduce the validity of epidemiological and clinical studies, and may also affect the results of **Paper I**. This risk is the smallest for exposures and outcomes where measurements happen in a controlled environment, for example biochemical analyses or genotyping undergoing regular quality control, and the largest for exposures and outcomes that are more difficult to measure or validate, for example self-reported parameters as smoking and alcohol-intake (that are also potentially affected by a social desirability bias, which occurs when participants report lower rates of health hazardous behavior to be perceived as more socially acceptable). Measurement error has the potential to both under- and overestimate the true association between exposure and outcome, and measurement error in confounders or covariates complicate the relationship further, because of an incomplete adjustment [@groenwold2020]. In **Paper 1**, the exposure variable in the genetic analyses is the *SLC5A2* genotype, where we expect a low rate of misclassification, based on the variant meeting Hardy-Weinberg expectations in the Copenhagen City Heart Study, Copenhagen General Population Study, and the UK Biobank. The only covariates used in the genetic analyses for risk estimation are age and sex, which are, for practical purposes, not prone to misclassification, as these are matched with the Danish Civil Registration system.\
For observational and mediation analyses, measurement error in the potential mediators might affect the results; however, we expect potential measurement error to be non-differential, meaning that it is not related to the exposure (genotype)[@ioannidis2008].

## National registries in Paper I

Endpoint data for the Copenhagen studies was collected using the national Danish Patient Registry and the Danish Registry of Causes of Death. The Danish Civil Registration System enabled individual-level data from the Copenhagen studies and the registries to be linked. Together these registries play a crucial role in facilitating epidemiological research in Denmark[@laugesen2023].\
The UK Biobank links the collected data to electronic health records from general physicians, hospital inpatient records, and death registries.

### The Danish Civil Registration System

The Danish Civil Registration System, established in 1968, contains personal information on all individuals residing in Denmark. Each person is assigned a unique, personal identification number (CPR-number) at birth or immigration, identifying the person across public and private institutions in Denmark. The registry includes details such as name, gender, date and place of birth, residence, citizenship, vital status, and CPR-numbers of parents and spouses[@pedersen2011]. This registry is updated daily and serves as a crucial tool for linking individual-level information on vital status and disease occurrence across various national Danish registries[@schmidt2014].

### The National Danish Patient Registry

The National Danish Patient Registry, established in 1977, is one of the oldest nationwide hospital registries and contains information on all hospital admissions in Denmark. From 1995 the coverage of the registry was expanded to also include outpatient and emergency room activities, and from 2003 also admissions to private hospitals and outpatient visits at public hospitals. The registry provides information such as dates of hospital admission and discharge, clinical data including diagnoses and surgical procedures. Diagnoses are classified according to the International Classification of Diseases, with ICD8 used until 1993 and ICD10 from 1994 onwards. During a hospital admission, the main disease for which the person is admitted is recorded as the primary diagnosis, while other concomitant diseases are recorded as secondary diagnoses[@schmidt2015].

### The Danish Registry of Causes of Death

The Danish Registry of Causes of Death, established in 1970, contains data on the date and cause(s) of death for all deaths in Denmark. The registry adheres to ICD8 until 1993 and ICD10 from 1994 onwards. Physicians record both the primary cause of death and any contributory causes[@helweg-larsen2011].

### UK Biobank electronic health records

Electronic health records in the UK Biobank comes from primary care records and inpatient hospital records.

Primary care data is collected through various data suppliers and other intermediaries, including the main primary care computer system suppliers in England, to collect primary care data for participants in the UK Biobank, who have provided written consent for linkage to health records. Primary care data is currently available for approximately 45% of the UK Biobank cohort (\~230,000 participants). Only data recorded by health care professionals working at general practices is included in the primary care data in the UK Biobank[@ukbioba2019].

Inpatient hospital admissions data for England, Wales, and Scotland is provided to the UK Biobank by different services managed by local subdivisions of the National Health Service (NHS). The data structures for England, Wales and Scotland differ with respect to data collection, organization, and validation. In all three countries, admitted patients are assigned one main (primary) code as well as a variable number of secondary codes. Differences in data structure between countries may lead to variation in the likelihood of non-primary conditions being coded and to differences in how hospital admissions are coded in the three countries. All of the current UK Biobank linked English and Welsh hospital data are coded in ICD10, but as data collection began earlier in Scotland (in 1981), the earlier Scottish data (before 1997) is coded in ICD9[@ukbioba2023a]. For **Paper I** only inpatient hospital data was used to define the endpoints.

### UK Biobank death registries

Mortality data in the UK Biobank is currently received from NHS England for participants in England and Wales and from the NHS Central Register, part of the National Records of Scotland, for participants in Scotland, and is updated quarterly. Death registry data includes date of death and the primary and contributory causes of death, coded using the ICD10 system[@ukbioba2023; @mortalit]

### FinnGen registers

FinnGen registry data used in **Paper 1** was collected and processed from the following different national health registers: hospital and outpatient visits in The Care Register for Health Care (HILMO, ICD8, ICD9 and ICD10), the Register of Primary Health Care visits (AvoHILMO, primary and secondary diagnoses using ICD10), the Cause of Death register (containing information on immediate, underlying and contributing causes of death using ICD8, ICD9 and ICD10 codes), and reimbursed medication entitlements and prescribed medicine purchases[@kurki2023].

## Study endpoints

### Paper I

An overview of the World Health Organization's International Classification of Diseases (ICD) codes, KELA (the Social Insurance Institution of Finland) reimbursement codes and Anatomical Therapeutic Chemical Classification System (ATC) codes for medicine purchases used to define endpoints in **Paper I** can be found in @tbl-icd.

#### Copenhagen City Heart Study and the Copenhagen General Population Study

Endpoints were defined according to ICD codes, and were heart failure, including all subtypes of heart failure, myocardial infarction, ischemic heart disease, ischemic stroke, cardiovascular mortality (primary cause of death: heart failure, myocardial infarction, ischemic heart disease, ischemic stroke, or peripheral arterial disease), and all-cause mortality, and were collected from January 1^st^ 1977, through December 13^th^ 2018, by reviewing diagnoses from all hospital admissions and outpatient visits in the national Danish Patient Registry[@schmidt2015] and the national Danish Registry of Causes of Death [@helweg-larsen2011], and any deaths from the national Danish Civil Registration System[@pedersen2011]. For the heart failure endpoint, this method makes the endpoint definition similar to the "hospitalization for heart failure" endpoint in cardiovascular outcome trials of SGLT2 inhibitors.

#### UK Biobank

Endpoints were defined according to ICD codes and were heart failure including all subtypes of heart failure), myocardial infarction, and ischemic heart disease, ischemic stroke, and cardiovascular mortality (primary cause of death: heart failure, myocardial infarction, ischemic heart disease, ischemic stroke, or peripheral arterial disease). Diagnoses were collected from the Hospital Inpatient Data field, which links to inpatient hospital data registries from England, Wales, and Scotland. The latest update of the diagnoses for heart failure, myocardial infarction, and ischemic heart disease from the Hospital Inpatient Data field was September 30, 2021. The latest update for all-cause mortality was November 12, 2021.

#### FinnGen

Information on endpoints was obtained from publicly available summary data, downloaded on March 17, 2023, and endpoints were defined according to ICD codes, KELA reimbursement codes, and ATC codes for medicine purchases. Endpoints were heart failure (I9_HEARTFAIL_ALLCAUSE, including all subtypes of heart failure, myocardial infarction (19_MI_STRICT), and ischemic heart disease (I9_IHD), embolic stroke (I9_STR_EMBOLIC), cardiovascular mortality (I9_K_CARDIAC), and all-cause mortality (DEATH)[@risteys].

\pagebreak

+------------------------------+-----------------------------------------------------------------------------+-----------------------------------------------------------------+----------------------------------------------------------------------------+
|                              | The Copenhagen City Heart Study and the Copenhagen General Population Study | The UK Biobank                                                  | FinnGen                                                                    |
+==============================+=============================================================================+=================================================================+============================================================================+
| **Heart failure**            | ICD8: 427.09-427.11                                                         | ICD9: 428.0-428.4, 4989                                         | ICD8 427.00, 427.10, 428, 782.4                                            |
|                              |                                                                             |                                                                 |                                                                            |
|                              | ICD10: I50.0-I50.9                                                          | ICD10: I50.0-I50.9                                              | ICD9: 402.9B, 428                                                          |
|                              |                                                                             |                                                                 |                                                                            |
|                              |                                                                             |                                                                 | ICD10: I11.0, I13.0, I13.2, I50.0                                          |
|                              |                                                                             |                                                                 |                                                                            |
|                              |                                                                             |                                                                 | KELA code: 201                                                             |
|                              |                                                                             |                                                                 |                                                                            |
|                              |                                                                             |                                                                 | ATC codes: C03CA01, C03EB01                                                |
+------------------------------+-----------------------------------------------------------------------------+-----------------------------------------------------------------+----------------------------------------------------------------------------+
| **Myocardial infarction**    | ICD8: 410                                                                   | ICD9: 410.9                                                     | ICD8: 410                                                                  |
|                              |                                                                             |                                                                 |                                                                            |
|                              | ICD10: I21-I22                                                              | ICD10: I21-I22                                                  | ICD9: 410                                                                  |
|                              |                                                                             |                                                                 |                                                                            |
|                              |                                                                             |                                                                 | ICD10: I21-I22                                                             |
+------------------------------+-----------------------------------------------------------------------------+-----------------------------------------------------------------+----------------------------------------------------------------------------+
| **Ischemic heart disease**   | ICD8: 410-414                                                               | ICD9: 410.9, 412, 413.9, 414.0 - 414.4                          | ICD8: 410-414, 798                                                         |
|                              |                                                                             |                                                                 |                                                                            |
|                              | ICD10: I20-I25                                                              | ICD10: I20-I25                                                  | ICD9: 410-414, 798                                                         |
|                              |                                                                             |                                                                 |                                                                            |
|                              |                                                                             |                                                                 | ICD10: I20-I25, I46, R96, R98                                              |
|                              |                                                                             |                                                                 |                                                                            |
|                              |                                                                             |                                                                 | KELA code: 206                                                             |
+------------------------------+-----------------------------------------------------------------------------+-----------------------------------------------------------------+----------------------------------------------------------------------------+
| **Ischemic stroke**          | ICD8: 410-414                                                               | ICD9: 434                                                       | ICD8: 434, 434.99                                                          |
|                              |                                                                             |                                                                 |                                                                            |
|                              | ICD10: I20-I25                                                              | ICD10: I63                                                      | ICD9: 434.1A                                                               |
|                              |                                                                             |                                                                 |                                                                            |
|                              |                                                                             |                                                                 | ICD10: I63.1                                                               |
+------------------------------+-----------------------------------------------------------------------------+-----------------------------------------------------------------+----------------------------------------------------------------------------+
| **Cardiovascular mortality** | Death from:                                                                 | Death from:                                                     | Death from:                                                                |
|                              |                                                                             |                                                                 |                                                                            |
|                              | ICD8: 410-414, 427.09-427.11, 433-434, 440, 443.99, 445                     | ICD9: 410-414, 428, 434, 440, 443\                              | ICD8: 394.0-394.2, 798-799                                                 |
|                              |                                                                             | \                                                               |                                                                            |
|                              | ICD10: I50.0-I50.9, I20-I25, I63, I70, I739, E115, E105, 145                | ICD10: I50.0-I50.9,  I20-I25, I63, I70, I739, E115, E105, E145) | ICD9: 394.0-394.2, 798-799                                                 |
|                              |                                                                             |                                                                 |                                                                            |
|                              |                                                                             |                                                                 | ICD10: I00-I02, I05-I09, I10-I15, I20-I25, I26-I28, I30-I52, R96, R98, R99 |
+------------------------------+-----------------------------------------------------------------------------+-----------------------------------------------------------------+----------------------------------------------------------------------------+
| **All-cause mortality**      | Death from all causes                                                       | Death from all causes                                           | Death from all causes                                                      |
+------------------------------+-----------------------------------------------------------------------------+-----------------------------------------------------------------+----------------------------------------------------------------------------+

: Table of World Health Organization's International Classification of Diseases codes, KELA reimbursement codes, and ATC codes for the endpoints included in **Paper I**. {#tbl-icd}

#### Strengths and limitations of the study endpoints

Misclassification of diagnoses used as endpoints might influence the results in **Paper I**. Misclassification occurs when an individual is put in the wrong group, for example if a patient with heart failure does not receive a health record heart failure diagnosis, or if a healthy individual erroneously receives a diagnosis. Misclassification can be a major issue in epidemiological research, however, most of the endpoints defined in **Paper I** are generally viewed as "hard" endpoints, meaning that it is unlikely that a myocardial infarction or a stroke patient is not seen in a hospital setting, or that a dead person is not registered as dead. Diagnoses related to heart failure and ischemic heart disease are more likely to be misclassified because of underestimation, as it is more common to be followed for these conditions in a general practice setting, particularly in the early stages of the diseases. Hard endpoints or not, there is a substantial risk of bias in studies that rely on a diagnosis of heart disease in a population that includes both male and female participants, since cardiovascular disease remain under-recognized, underdiagnosed, and undertreated in women compared to men[@vogel2021; @burgess2022]. This may lead to underestimation of the risk of heart failure in the population.

As diagnoses in the Danish registries are used to define the endpoints, the validity of the registry data is important. In validation studies of the national Danish Patient Registry, the positive predictive value (PPV) for heart failure was between 84% and 95% with the highest value for diagnoses given at a cardiology department[@delekta2018]. For myocardial infarction it has been reported to be 97%[@sundbøll2016], whereas other ischemic heart diseases were reported to have lower PPVs (46-62% for angina pectoris and 87-96% for acute coronary syndrome including unstable angina and myocardial infarction) or have not been validated[@bork2017]. The validity of ischemic stroke diagnoses is also high with reported PPVs between 88% and 97%; however, it is expected that many ischemic strokes have been misclassified as other stroke types or as unspecified stroke [@krarup2007; @johnsen2002], which may lead to underestimation of the true association. In both Copenhagen Studies, ischemic stroke diagnoses were validated individually[@truelsen2002], supporting the validity of the endpoints.\
In general, diagnoses with lower PPVs should be used with caution in epidemiological studies, if validation is not feasible.\
The Danish Civil Registration System provides highly valid all-cause mortality outcomes, as every death in Denmark is confirmed by a medical doctor, and the registry is updated on a daily basis. However, the validity of cause-specific mortality based on diagnoses from the national Danish Registry of Causes of Death is less reliable. This is because of changes in the registration of death certificates over the years, lack of regular validation of causes, and autopsy rates below 10%[@helweg-larsen2011]. It is expected that there may be misclassification of some cardiovascular deaths, that may lead to either over- or underestimation of the true association.\
Although the point of ICD codes is a uniform way to report disease worldwide, it is expected that there may be regional differences in how diagnoses and deaths are registered, limiting the comparability between studies from different countries.

### Paper II

#### Meta-analysis endpoints

Endpoints for the meta-analysis in **Paper II** were changes in total, LDL, and HDL cholesterol and triglycerides. Generally, these were not primary endpoints in the included randomized controlled trials, and information on sample collection and analysis methods was sparse or non-existing. However, since we investigate change from baseline to follow-up in the selected endpoints, we do not suspect differences in sample collection and analysis methods to have a major influence on the results since these are less important in change measures than in absolute measures.

## The 'genetic variation as proxy for drug effects' study design in Paper I

#### Mendelian randomization and genetic variation as proxy for drug effects

Drug effects can be explored by investigating the natural genetic variation in their target genes, a method closely linked to the theoretically similar method of Mendelian randomization[@benn2018] (@fig-randomization). Both methods are based on the idea of using genetic variation as a proxy for a certain exposure to measure an unconfounded effect on an outcome. Additional advantages of these genetic approaches include the absence of reverse causation, which is a limitation in observational epidemiology, the ability to predict long-term effects of a treatment directly in humans[@stender2016], and the possibility to examine effects specific to one pathway, unlike in randomized controlled trials of drugs where concomitant medication may confound the association.

![The principle of Mendelian randomization (right) in comparison to a randomized controlled trial (left) that form the basis of the study of drug effects using genetic variants. Image created with BioRender.](randomization.png){#fig-randomization}

In Mendelian randomization, and in the study of drug effects using the same basic principle, genetic variants are used as proxies for a modifiable risk factor or exposure to estimate the effect of changes in the modifiable risk factor on a certain outcome (@fig-MR).

![Mendelian randomization design. The analysis is done in 4 steps: 1) the observational association between exposure (X) and outcome (Y) is determined, 2) the association between the genetic instrument (Z) and the exposure (X) is determined, 3) the association between the genetic instrument (Z) and the outcome (Y) is determined, and 4) instrumental variable analysis is performed to assess the causal effect of the exposure (X) on the outcome (Y)[@benn2018].](MR.png){#fig-MR}

We aimed to apply these principles to examine the hypothesis that genetic variation in *SLC5A2* would mimic pharmacological SGLT2 inhibition, and that this would be supported by 1) the knowledge from randomized controlled trials that SGLT2 inhibitors reduce risk of cardiovascular disease and mortality (similar to arrow 1 in @fig-MR), 2) the similarity between a genetic variant known to cause familial renal glucosuria, a phenotype resembling that of pharmacological SGLT2 inhibition, and pharmacological SGLT2 inhibition (similar to arrow 2 in @fig-MR), and 3) a similar effect of the genetic variant on the outcomes of interest; risk of cardiovascular disease and mortality, as pharmacological SGLT2 inhibition (similar to arrow 3 in @fig-MR). With a genetic proxy for pharmacological SGLT2 inhibition, we would then be able to examine potential mediators between SGLT2 inhibition/SGLT2 genetic variation and risk of cardiovascular disease and mortality (@fig-genetic_proxy). This is different from a Mendelian randomization study which also includes an instrumental variable analysis (@fig-MR)

![In **Paper 1**, we hypothesized that genetic variation in *SLC5A2* would mimic pharmacological SGLT2 inhibition and that this would allow us to examine potential mediators (M) between the exposure SGLT2 inhibition/SGLT2 genetic variation (X) and the outcome risk of cardiovascular disease and mortality (Y).](genetic_proxy.png){#fig-genetic_proxy}

#### Limitations of the genetic proxy design

Several limitations of the genetic proxy design may have influenced our results. These include:

-   **Non-random assortment of alleles**, which occurs if genetic variants are indeed not randomly allocated as they should be for the study design (@fig-MR) to be tenable. This could occur as a result of non-random mating[@border2022] or perhaps because of sex-specific participation bias in the population[@pirastu2021], and may cause genes to be inherited together, potentially giving rise to pleiotropy (see below).

-   **Weak instrument bias** may occur if the genetic instrument only explains part of the variation in the exposure variable. In formal Mendelian randomization, where one or more genetic instruments are used as proxies for a measurable exposure variable, for example plasma glucose or LDL cholesterol, the strength of the association between instrument and exposure may be estimated by the F statistic from the regression of one or more genetic variants on the exposure variable[@benn2018]. One of the challenges in using genetic variants as proxies for drug treatment, is that this association is not easily quantified in a regression, because a drug may have several effects on measurable variables. In the case of **Paper I**, ideally, we would have been able to assess the strength of the instrument by the association between genetic variant and urinary glucose, however, this was not possible in either of the used study populations. From other studies, we found that individuals heterozygous and compound heterozygous for the *SLC5A2* rs61742739 genetic variant had daily urine excretions of 3-12g[@calado2004; @kleta2004; @calado2008] which are modest compared to the daily 60-80g urine glucose excretion observed with pharmacological SGLT2 inhibition[@abdul-ghani2015], potentially indicating a weak genetic instrument. We do, however, argue that life-long effects of genetically increased urine glucose excretion may yet be important.

-   **Pleiotropy** may occur if the genetic instrument is not only associated with the exposure but also other variables, and can be divided into vertical and horizontal pleiotropy[@benn2018]. Vertical pleiotropy is when a genetic variant is associated with several risk factors on the same biological pathway from the variant to the outcome of interest and does not cause results to be invalidated. Horizontal pleiotropy is when a genetic variant is associated with other confounding pathways also causal for the outcome. Horizontal pleiotropy can potentially bias the estimate of the association between exposure and outcome. To minimize the risk of pleiotropic effects, we selected a genetic variant in the gene that encodes SGLT2, known to alter an amino acid in the finished protein, adding biological plausibility to the association between genetic variant and SGLT2 function rather than to other horizontal pleiotropic effects.

-   **Population stratification** may occur if there is genetic heterogeneity in the study population, that divides the population into subgroups, for example different ethnicities with different genetic compositions, or if there is non-random mating within the population[@border2022]. In some instances, population stratification may be identified by Hardy-Weinberg disequilibrium, indicating an unexpected distribution of alleles in the population. Hardy-Weinberg disequilibrium is, however, most often caused by genotyping errors and this reason must be excluded before further analyses. We observed no deviations from Hardy-Weinberg expectations in the Copenhagen City Heart Study, the Copenhagen General Population Study, or the UK Biobank.

-   **Canalization** is a developmental compensation mechanism that may theoretically cause the effects of the *SLC5A2* rs61742739 genotype on potential mediators to be diminished through alternative, compensatory pathways seeking to normalize the effect of the variant[@hallgrimsson2019]. The mechanisms behind canalization are poorly understood, and there is no established methodological technique to assess possible canalization. Canalization is a potential cause for confounding in the prediction of phenotypic effects in genetic studies.

## Statistical analysis

R (version 4.1.2) was used for all statistical analyses in the two papers.

### Comparisons between groups

Hardy-Weinberg equilibrium for the *SLC5A2* rs61742739 genotype in the Copenhagen City Heart Study, the Copenhagen General Population Study, and in the UK Biobank was tested using the Pearson 𝜒2 test.

### Prospective risk estimates

Cox proportional hazards regression models were used to estimate the prospective risk, or hazard ratios, of disease outcomes in the Copenhagen City Heart Study and the Copenhagen General Population Study as a function of *SLC5A2* rs61742739 genotype.\
The Cox proportional hazards regression compares time to event or outcome across exposure groups and is based on the proportional hazards assumption, assuming that the ratio of hazards between different exposure groups remain constant over time. This assumption may be violated if there is an interaction between time and the exposure variable. Depending on the study design, different follow-up definitions may be used in the model. In a cardiovascular outcome trial of SGLT2 inhibitors, it would for example be reasonable to investigate time from randomization or treatment initiation to event, supposing that treatment with SGLT2 inhibitor or placebo was the main exposure of interest. In this case, the risk sets would consider individuals at risk after treatment initiation[@kirkwood2003]. However, when using genetic variation as the main exposure, it may be more suitable to construct the risk sets with follow-up from time since birth, as genotypes remain constant through life. Since participants were not followed from birth, models were left-truncated (because of delayed entry). For the analyses with the rs61742739 variant as exposure, follow-up therefore began at the earliest possible date for each participant, which was either the 20^th^ birthday or the beginning of the national Danish Patient Register (January 1^st^, 1977), whichever came last. Also, different time scales may be used in the model. Using age as the time scale means that risk estimates are based on the fact that participants with an event are compared with other participants of the same age, and in **Paper I**, age was therefore used as the time scale for all Cox regressions.\
None of the included individuals had a diagnosis of either outcome prior to entry date. Follow-up ended on the day of censoring due to an event, emigration, or on December 13, 2018, whichever came first for the relevant analyses. The genetic Cox proportional hazards regressions were adjusted for sex and date of birth, to account for differences in recruitment, diagnosis, and treatment according to sex and over time.\
Furthermore, the associations between glucose, potassium, and lipoprotein(a) in the Copenhagen City Heart Study + the Copenhagen General Population Study and HbA1c in the UK biobank, and the risk of cardiovascular disease and all-cause mortality were examined using Cox proportional hazards regressions. These analyses were adjusted for age, sex, date of birth, body mass index, systolic and diastolic blood pressure, LDL-cholesterol, HDL-cholesterol, plasma triglycerides, and smoking status to account for known confounders of cardiovascular disease. Follow-up began on the day of examination, end ended as described above for the genetic models. None of the included individuals had a diagnosis of either of the outcomes prior to entry date. For the UK Biobank, follow-up ended on date of censoring due to cardiovascular disease on September 30, 2021, and due to death on November 12, 2021, whichever came first for the relevant analyses.

The proportional hazards assumptions for all Cox regressions were tested for independence between residuals and time by correlating the corresponding scaled Schoenfeld residuals with time. No violations of the proportional hazards assumption was found, supported by non-significant relationships for all endpoints and by visual inspection of plots of the scaled Schoenfeld residuals against time.

### Cross-sectional risk estimates

Logistic regression models were used to calculate cross-sectional risk estimates, or odds ratios, for the purpose of comparison and meta-analysis of odds ratios of disease outcomes between the Copenhagen City Heart Study + the Copenhagen General Population Study, the UK Biobank, and FinnGen. This was necessary, as we did not have access to individual level data in FinnGen that would otherwise have allowed comparison and meta-analysis of hazard ratios. Odds ratios and hazard ratios are often used in different study types and have different interpretations. Compared to odds ratios, hazard ratios include time-to-event and the ability to use age as the time scale, and whereas odds ratios deal with determining associations between exposure and outcome, hazard ratios deal with event rates, providing information on how one exposure group changes relatively to another[@george].\
Logistic regression models were adjusted for sex and date of birth in the Copenhagen City Heart Study + the Copenhagen General Population Study, and the UK Biobank, and summary statistics from FinnGen were adjusted for sex, age, 10 principal components, and genotyping batch[@kurki2023].

### Mediation analysis

Mediation analysis was used to determine the proportions of the total effect of *SLC5A2* rs61742739 genotype on risks of cardiovascular disease and all-cause mortality mediated by plasma glucose, hemoglobin A1c (in the UK Biobank only), and lipoprotein(a) in the Copenhagen City Heart Study + the Copenhagen General Population Study, and the UK Biobank. One of the goals in **Paper I** was understanding the process from genetic variation to altered risk profile (@fig-genetic_proxy) and to examine potential mediators of the effect, made possible by mediation analysis.\
Mediation analysis standard procedures involve a set of linear regression models from which the mediation effects are estimated[@baron1986; @mackinnon2007]. There are, however, several limitations to this method, especially once linear regressions fail to describe the associations, or if the purpose of the mediation analysis is to study causal mechanisms where certain assumptions must be met[@imai2010]. Since we aimed to study the mediated proportions of a relationship estimated through Cox proportional hazards regression or logistic regression, we selected the R mediation package[@tingley2014] that allows the use of different types of data and statistical methods. For practical coding purposes, mediation analyses were fitted with linear regression models to describe the relationship between genetic variant and potential mediator and logistic regression models to describe the relationship between genetic variant and outcome. Models were adjusted for age at examination and sex, and the logistic regressions further adjusted for date of birth. Potential mediators were tested separately in mediation analyses.

### Meta-analysis

Meta-analysis was used in both **Paper I & II** to aggregate results from different studies. In **Paper I**, we performed a meta-analysis of odds ratios of cardiovascular outcomes and mortality in the Copenhagen City Heart Study + the Copenhagen General Population Study, the UK Biobank, and FinnGen to validate our results from the two Copenhagen studies in other cohorts of different compositions and ethnicities. We fitted both random and fixed effects models although we expected substantial heterogeneity based on our knowledge of the composition of the different cohorts (see "Study populations and materials"), supporting the use of a random effects model. In a fixed effects model, it is assumed that the observed variation in effects in the different studies is due to sampling variation only, and that the underlying effect - in this case of genetic variation - is the same (or fixed) in all studies. The individual weight for each study is estimated using the inverse variance of the estimated log odds ratio. Compared to the fixed effects model, the random effects model allows for the heterogeneity between studies to be incorporated into the model, by assuming a normal distribution of the "true" log odds ratios in the included studies. The between-study variance is used to modify the weights of the included studies and then calculate the random effects summary estimates[@kirkwood2003]. Heterogeneity was assessed using the I^2^s from the random effects models.\
In **Paper II**, meta-analysis was the main statistical method used. Here, also random and fixed effects models were both used to estimate the effect of SGLT2 inhibitor treatment on change in lipid and lipoprotein levels, even though we also here expected a high level of heterogeneity because of differences in study populations, lengths, and SGLT2 inhibitor types. Both a full analysis of all included studies and subgroup analyses for SGLT2 inhibitor types, treatment dose, and non-Asian/Asian ethnicity were calculated to examine the effects. Meta-regression was used to determine the effect of treatment dose, lipid-lowering therapy, and HbA1c on meta-analysis estimates, and was performed by including these as a moderators in the random effects models. Furthermore, meta-regression was done to investigate the heterogeneity accounted for by differences in treatment dose.\
Meta-analysis results may be limited by different biases (described in the "Study populations and materials, Paper II" section). Funnel plots, Egger's test, and Begg and Mazumdar's rank test were used to screen for publication bias of which we found no indication. Sensitivity analyses were done for each endpoint using the leave-one-out method, which did not alter the results.

All meta-analyses in the included papers were done using the R metafor package[@viechtbauer2010].

### Limitations in statistical analyses

As in the other steps of the methodological process of conducting a scientific study, the statistical analyses may give rise to limitations. These can be due to errors in statistical design, analysis, interpretation, and reporting. While there are many such pitfalls, only a few points, relevant to the two included studies, will be mentioned here.

#### Calculating power and estimating sample size

An important consideration when designing a study is to include a sufficient sample size to test the hypothesis. Failing to do this may lead to type 1 errors (false positive or rejecting a null hypothesis that is in fact true) or type 2 errors (false negative or failing to reject a null hypothesis that is in fact false)[@jones2003].\
For the Copenhagen City Heart Study and the Copenhagen General Population Study power calculations were done before genotyping to ensure a sufficient sample size and power in the study under the given circumstances. Power calculations were done with the expectancy to include the full population (N \~ 110 000) for the study endpoints heart failure and all-cause mortality with population prevalences of 0.068 and 0.28, respectively. The *SLC5A2* rs61742739 variant had a reported allele frequency of \~0.01[@rs617427], and power calculations were done for this frequency, but also for alternative scenarios of minor allele frequencies of 0.005 and 0.04, which is the highest reported minor allele frequency in any population (@fig-power). For heart failure, with a population prevalence of 0.068 and a minor allele frequency of 0.01, we had 80% power to detect a hazard ratio of less than 0.77 or above 1.24 for expected differences in risk between carriers(AG/GG) and non-carriers(AA) of the *SLC5A2* rs61742739 genetic variant. For all-cause mortality, with a frequency of 0.28 in the population and a minor allele frequency of 0.01, we had 80% power to detect a hazard ratio of less than 0.88 or above 1.12. Power calculations were done with the R package survSNP[@owzar2012].

![Power calculations for the Copenhagen City Heart Study and the Copenhagen General Population Study (N = 110 000) for genetic variants with minor allele frequencies, denoted "q" in the plot, of 0.005, 0.01 (corresponding to the minor allele frequency of the SLC5A2 rs61742739 variant), and 0.04 (corresponding to the highest minor allele frequency reported for the SLC5A2 rs61742739 variant). Panel A shows the curve for power as a function of hazard ratio for heart failure with an population prevalence of 0.068. Panel B shows the curve for power as a function of hazard ratio for all-cause mortality with an population prevalence of 0.28.](power.png){#fig-power}

#### Multiple testing

Multiple testing refers to any occasion where more than one hypothesis is tested at the same time. If conclusions are drawn regarding the individual hypotheses based on the unadjusted marginal p-values, there is often a considerable probability that some of the true null-hypotheses will be rejected by chance[@higdon2013; @george2016]. If 100 hypotheses of a relationship between exposure and outcome, all of them being true, are tested simultaneously and the significance level is set to 0.05, the expectation would be that five true hypotheses would be rejected (type 1 error). If all hypotheses were mutually independent, the probability that one true null hypothesis would be rejected would be given by 1 - 0.95^100^ = 0.994.\
The problem of multiple testing arises if multiple hypotheses are tested at the same time and the list of p-values is screened for significant results after the analyses have been carried out.\
A classical method to control for the error rate is the Bonferroni correction where the corrected significance level is based on the formal significance level (often 0.05) divided by the number of hypotheses tested[@haynes2013]. Following the example above, the Bonferroni adjusted significance level would be 0.05/100 = 0.0005, meaning that only tests with p-values ≤ 0.0005 should be interpreted as significant. The Bonferroni method is an easily applied test, that works well in settings involving a few tests. However, in other settings, for example Genome Wide Association Studies where thousands or millions of hypotheses are tested at once, other methods must be considered[@rice2008]\
In **Paper I**, multiple testing is performed both in establishing the association between the *SLC5A2* rs61742739 genetic variant and the six endpoints on cardiovascular disease and mortality, in testing the association of the variant and 13 different potential mediators, and in the mediation analysis of three different potential mediators for 4 different endpoints (=12 hypotheses), which is a limitation to the study. If Bonferroni corrections were applied to each step, results would be slightly different: with regard to the endpoints, only the associations between the *SLC5A2* rs61742739 genetic variant and heart failure, ischemic heart disease and all-cause mortality (and not myocardial infarction) would pass the significance level; with regard to the potential mediators, only glucose in the Copenhagen studies and blood hematocrit in the UK Biobank would be deemed significant, and in the mediation analyses glucose would only be a significant mediator in the association between the *SLC5A2* rs61742739 genetic variant and heart failure. This does, however, not change the overall conclusion of the study, namely, that the lower risks of cardiovascular disease and mortality by the *SLC5A2* rs61742739 genetic variant were at most minimally mediated through lower plasma glucose. Importantly, the interpretation of a scientific study should never rely on p-values or significant results only, although, these have historically been and are still often ascribed an immense importance.

#### Statistical errors in meta-analysis

Statistical errors in meta-analyses are common[@gøtzsche2007; @kadlec2023] and often due to errors in calculations, faulty imputations of missing data[@george2016], confusion between standard error and standard deviation[@gøtzsche2007; @kadlec2023], failure to account for correlated observations, and analysis of within-group rather than between-group (especially in meta-analyses of randomized controlled trials)[@kadlec2023]. It is generally advised that meta-analyses and systematic reviews should be undertaken by a team where at least some team members are experienced in the methods, thereby limiting the chance of committing the above-mentioned errors. In the meta-analyses in **Paper II**, several steps in the process, vulnerable to error, was undertaken only by the first author. Although measures were taken to ensure low levels of error, such as screening abstracts twice, double checking data extraction, following guidelines in the Cochrane Handbook[@cochrane], considering the above mentioned common sources of error, and studying the statistical software, this is a limitation to the study. However, the results of the meta-analysis are similar to previously published results of similar meta-analyses[@sánchez-garcía2020; @cai2018].

\pagebreak
